src/main/java/org/opentripplanner/routing/core/RoutingContext.java

```diff
@@ -237,14 +237,14 @@ public void checkIfVerticesFound() {
         List<RoutingError> routingErrors = new ArrayList<>();
 
         // check origin present when not doing an arrive-by batch search
-        if (fromVertices == null) {
+        if (fromVertices == null && !(opt.oneToMany == true && opt.arriveBy == true)) {
```

Comment:
I think this check was simplified, because we do not allow one-to-many search any more, just in the access/egress search - but this code should not be part of that?

Reply:
This code is needed for the AStar search in StopFinder